lms/templates/courseware/progress_graph.js

```diff
@@ -287,30 +289,40 @@ $(function () {
               edx.HtmlUtils.HTML('<div class="overallGrade" style="position:absolute;left:' + (o.left - 12) + 'px;top:' + (o.top - 20) + 'px">'),
               edx.HtmlUtils.HTML('<span class=sr>'),
               gettext('Overall Score'),
-              edx.HtmlUtils.HTML('</span>'),
+              edx.HtmlUtils.HTML('<br></span>'),
               '${'{totalscore:.0%}'.format(totalscore=totalScore) | n, js_escaped_string}',
               edx.HtmlUtils.HTML('</div>')
           )
       );
 
     %endif
+
+    $grade_detail_graph.find('.xAxis .tickLabel').attr('tabindex', '0').focus(function(event) {
+        var $target = $(event.target), srElements = $target.find('.sr'), srText="", i;
+        if (srElements.length > 0) {
+            for (i = 0; i < srElements.length; i++) {
+                srText += srElements[i].innerHTML;
+            }
+            // Position the tooltip slightly above the tick label.
+            showTooltip($target.offset().left - 70, $target.offset().top - 120, srText);
+        }
+    });
+
+    $grade_detail_graph.focusout(function(){
+        $("#tooltip").remove();
+    })
   }
   
       
   var previousPoint = null;
   $grade_detail_graph.bind("plothover", function (event, pos, item) {
-    $("#x").text(pos.x.toFixed(2));
```

Comment:
Don't see any reason why these are set (if such an element even exists on the page, which I doubt).